### PR TITLE
Add AmericasNLP 2022 baselines

### PR DESCRIPTION
**Pull request by akreal**
This is the baseline setup for the ASR task of [the second AmericasNLP competition](http://turing.iimas.unam.mx/americasnlp/st.html) that I help to organize. We would be happy if it could be merged and attract some submissions from the ESPnet community.

Please let me know if there are any questions or comments.

### CODE REVIEW

**Comment by sw005320** [comment]
LGTM.
My comment is that the WER performance is not great.
The CER is in a reasonable range, so I'm expecting that the word recognition (boundary or compound words or not sure) of these languages is challenging.
We may only use the CER for this task.

Can you paste an example of the WER result?
I want to understand what happens.

**Comment by akreal** [comment]
Thank you for the comment @sw005320 !

Sometimes it indeed looks like a word recognition problem:
```
Speaker sentences 153:  waikhana001572.wav   #utts: 1
id: (waikhana001572.wav-waikhana001572.wav)
Scores: (#C #S #D #I) 0 1 3 0
REF:  SAA IHIG-UTAHA ALII TAHA
HYP:  *** ********** **** SAGUTA'LITHA
Eval: D   D          D    S
```

Sometimes something else might be wrong:
```
Speaker sentences 201:  waikhana001620.wav   #utts: 1
id: (waikhana001620.wav-waikhana001620.wav)
Scores: (#C #S #D #I) 0 5 0 22
REF:  ** * * * * * * * * * * * * * * * * * * * * * YU'U MANILE SEGUNDA-FEIRA MANI IHIMAHAPE
HYP:  IU A A A A A A A A A A I I I I I I I I A A A A    A      A             U    OMU
Eval: I  I I I I I I I I I I I I I I I I I I I I I S    S      S             S    S
```

Here are the WER results:
 - [Bribri](https://github.com/espnet/espnet/files/8848773/bzd_result.txt) (WER 92.3%);
 - [Guarani](https://github.com/espnet/espnet/files/8848774/gug_result.txt) (WER 101.0%);
 - [Kotiria](https://github.com/espnet/espnet/files/8848775/gvc_result.txt) (WER 94.2%);
 - [Quechua](https://github.com/espnet/espnet/files/8848776/qwe_result.txt) (WER 85.6%);
 - [Wa'ikhana](https://github.com/espnet/espnet/files/8848777/tav_result.txt) (WER 114.0%);

Amount of training data is between 0.37 and 2.70 hours. This can partly explain the high WER.

File: egs2/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ See: https://espnet.github.io/espnet/espnet2_tutorial.html#recipes-using-espnet2
...
 | aishell3                | AISHELL3 Mandarin multi-speaker text-to-speech                                                                                   | TTS                     | CMN                   | https://www.openslr.org/93/                                                                                  |              |
 | aishell4                | AISHELL4 Open Source Mandarin Speech Corpus in Conference Scenario                                                               | ASR/SE                  | CMN                   | https://www.openslr.org/111/                                                                                 |              |
+| americasnlp22           | The Second AmericasNLP Competition                                                                                               | ASR                     | 5 languages       | http://turing.iimas.unam.mx/americasnlp/st.html                                                                  |              |
```
Comment (sw005320) [inline egs2/README.md:15]: Can you add a three-letter language code for these languages?

Comment (akreal) [reply egs2/README.md]: Done!

File: egs2/americasnlp22/asr1/run.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+asr_config=conf/train_asr_transformer.yaml
+inference_config=conf/decode_asr.yaml
+
+./asr.sh \
+    --lang ${lang} \
+    --ngpu 1 \
+    --nbpe 100 \
+    --use_lm false \
+    --max_wav_duration 38 \
```
Comment (ftshijt) [inline egs2/americasnlp22/asr1/run.sh:22]: May I ask why we add the argument?

**Review by ftshijt — APPROVED** [review]
LGTM! Many thanks for the contribution

**Pull request merged**